### ISSUE TITLE
Inside Movable Insulation causing increased heat loss

### ISSUE THREAD

**Issue by mjwitte** [NatLabRockies/EnergyPlus#10865]
Issue overview
--------------
User test file shows that inside movable insulation results are nowhere near equivalent results with the insulation layer included in the base construction. In fact, the heating load increases significantly with the inside movable insulation. Outside movable insulation results are reasonably close to the equivalent construction.

Test file results (run with USA_CO_Denver-Stapleton.724690_TMY.epw):
| Filename | Insulation Location | Method | District Heat | District Cool | % diff MI vs Constr,  Heating | Cooling |
|---|---|---|---|---|---|---|
| MovableIntInsulationLights-NoInsulation | None | | 28033 | 10345 | | |
| MovableIntInsulationLights-Outside-Construction | Outside | Construction | 14983 | 5836 | | |
| MovableIntInsulationLights-Outside-MovIns | Outside | MI | 14516 | 6040 |  -3% | +3.5% |
| MovableIntInsulationLights-Inside-Construction | Inside | Construction | 14263 | 6577 | | |
| MovableIntInsulationLights-Inside-MovIns | Inside | MI | 33583 | 10303 | +135% | +57% |

Note that the inside movable insulation case has a significantly higher heating load than the base case with no insulation.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus 24.2
 - [Unmethours 48835](https://unmethours.com/question/48835/issue-with-surfacecontrolmovableinsulation-for-wall-applications/)  and helpdesk ticket 16142.

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSupport\DefectFiles
 - [ ] Ticket added to EnergyPlus Defect Complexity (Github Project)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
Comparing the NoInsulation, Inside-Construction and Inside-MovIns, for surface `ZN001:WALL001`

![Image](https://github.com/user-attachments/assets/443a6886-8303-4950-8ee8-f8231f041af2)

**Comment by jmarrec** [comment]
![Image](https://github.com/user-attachments/assets/5fb18524-490d-44bf-9bf1-43c52b7ba792)

**Comment by mjwitte** [comment]
@RKStrand Any thoughts on this?

**Comment by RKStrand** [comment]
@mjwitte Yeah, something does not look right here.  I'll be working on defects in the near future, and this will be the first thing that I look at.  I've assigned it to myself.  Thanks!

**Comment by RKStrand** [comment]
Update:
I think that there might actually be two problems here.  First, as I was reviewing the heat balance equations for the movable insulation, I noticed that the short wavelength solar was being added between the surface and the movable insulation.  While that option exists for exterior movable insulation (for transparent insulation materials or TIM), when the interior movable insulation was added, it did not have the functionality of being transparent.  All interior movable insulation was supposed to be opaque according to the IO Ref.  I guess I have to take the blame for the error--must have made a mistake during the derivation of equations back when I implemented this.  I believe that the thinking was that interior movable insulation does not need to be transparent.  In any case, the equations can be fixed without too much trouble here though testing it might be a bit cumbersome.

Unfortunately, that is not actually the source of the problem here because this is a building with no windows so there is no short wavelength solar on the interior.  I think (could be wrong) that there is an issue with the temperatures that are being passed to the internal long wavelength radiation between surfaces distribution (CalcInteriorRadExchange).  What is being passed is SurfTempIn for all cases, and in all cases other than the case when movable insulation is present, that is correct.  In all other cases, SurfTempIn is simply set to SurfTempInTmp after SurfTempInTmp is calculated as the inside face surface temperature.  However, in the case of movable insulation, SurfTempIn is the temperature at the interface between the surface and the movable insulation (when present), and SurfTempInTmp is the temperature at the inside of the movable insulation (as in, the side facing the rest of the zone).  So, SurfTempInTmp is the variable that should be passes to CalcInteriorRadExchange when there is movable insulation, NOT SurfTempIn which is the temperature "behind" the movable insulation.  I still need to figure out what is the "best" way to approach this and then test it.  The code and equations are complex so part of the purpose of this update is to document what I think I'm finding because things get confusing very quickly with different variables and the heat balance equations for movable insulation.

**Comment by RKStrand** [comment]
Update (should have been posted early June 2025 but I moved on because this was going "nowhere"):
Fixing where the solar hits and which temperature is passed to the interior radiant exchange subroutine didn't seem to really make much difference.  Changes were slight and there is still a significant difference between the version which the insulation added to the construction and insulation added as an "always on" movable insulation.  I've gone back and looked at the governing equations and they seem to be correct as far as I can tell.  As @jmarrec posted, there is a significant temperature difference on the inside between the "cons" and the "movins" case.  In the last graph above, the red and orange lines should be on top of each other but they show a roughly 4C difference.  I suppose it is possible that the level of insulation being added causes a problem for the movable insulation case--as in, are there multiple solutions (like some cases we have noticed with extremely light (thermally) constructions) and this simply goes to the wrong solution in this case?  The case in the defect file has an insulation level that is around 5 times higher than the value for the wall itself.  So, I suppose something like this is possible.  In any case, I had to set this aside because it was not progressing and I needed to hit pause.  I'm not sure when I'll get back to this.  If someone else has ideas or wants to pick this up, feel free to do that and reassign it to yourself.
